src/Testcontainers.FirebirdSql/FirebirdSqlBuilder.cs

```diff
@@ -0,0 +1,153 @@
+namespace Testcontainers.FirebirdSql;
+
+/// <inheritdoc cref="ContainerBuilder{TBuilderEntity, TContainerEntity, TConfigurationEntity}" />
+[PublicAPI]
+public sealed class FirebirdSqlBuilder : ContainerBuilder<FirebirdSqlBuilder, FirebirdSqlContainer, FirebirdSqlConfiguration>
+{
+    public const string FirebirdSqlImage = "jacobalberty/firebird:v4.0";
+
+    public const ushort FirebirdSqlPort = 3050;
+
+    public const string DefaultDatabase = "test";
+
+    public const string DefaultUsername = "test";
+
+    public const string DefaultPassword = "test";
+
+    public const string DefaultSysdbaPassword = "masterkey";
+
+    private const string TestQueryString = "SELECT 1 FROM RDB$DATABASE;";
```

Comment:
We need to take a look at this statement. It fails for the version 2.5. While the exit code is `0` the error output contains:

    Statement failed, SQLSTATE = 08001
    I/O error during "open" operation for file "test"
    -Error while trying to open file
    -No such file or directory
    After line 0 in file /tmp/325ddf2a-2eff-4c45-af40-e80dd082d587/p3traelx.r1t
    Use CONNECT or CREATE DATABASE to specify a database
    Use CONNECT or CREATE DATABASE to specify a database

Reply:
I looked into this issue and indeed `/firebird/data/` is required to connect to the database for version 2.5 (isql CLI) too. Changing the database in the isql wait strategy command to `/firebird/data/test` works fine.

Reply:
I saw the changes you did, looks good to me now the way it is.